ibm/resource_ibm_kms_key.go

```diff
@@ -438,7 +428,6 @@ func resourceIBMKmsKeyRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("encrypted_nonce", key.EncryptedNonce)
 	d.Set("iv_value", key.IV)
 	d.Set("key_name", key.Name)
-	d.Set("crn", key.CRN)
```

Comment:
Why you removed the d.set of crn

Reply:
Ohh... Thats a mistake.. Thanks for pointing that out.